### ISSUE TITLE
Change email to username?

### ISSUE THREAD

**Issue by JaakkoLipsanen** [JaakkoLipsanen/evento#62]
So, we are not really using the email to anything and giving your email on registration is a bit more "intrusive" than just username. Maybe?

**Comment by anttilip** [comment]
Yeah it's fine by me. I guess email is needed only if we require confirmation etc, which will not be implemented at least for now.
